package.json

```diff
@@ -14,13 +14,17 @@
     "tslint": "tslint -c tslint.json -p tsconfig.json",
     "prettier": "prettier --write \"**/*.{js,ts}\"",
     "lint": "yarn prettier ; yarn tslint",
-    "test": "NODE_ENV=test jest --detectOpenHandles --forceExit --coverage --verbose"
+    "jest": "NODE_ENV=test jest --detectOpenHandles --forceExit --coverage --verbose",
+    "test": "yarn stop ; yarn start ; sleep 5 ; yarn jest ; yarn stop"
```

Comment:
Ideally, this should be a small node script that takes care of stoping/starting and running tests

Reply:
do you plan to run all the tests as integration tests? I think that's overkill. I could see this being useful for testing the reading/writing DB layer, but for business logic, you should mock the DB layer, imo.

Reply:
there is actually a node script that handles the starting and stopping. That's what `yarn start` and `yarn stop` invoke. I need to do it before and after tests though which is why this is here. It'd be better to do it in a setup file or something though. I'll do that.

The `sleep` here is also an odd thing. Ideally we'd want to just wait in our script until the service is ready. Since we're just running `execSync` though, I'm not really sure how we would do that. Regardless, I figure that could be an improvement made later.

Regarding your integration tests comment: In my opinion, it's the best way to test our graph traversals since a lot of the logic which needs to be tested lives in those traversals. If we mock gremlin our tests will be relatively useless. Since the DB we are interacting with is local, it feels more like a functional test to me, and I'm cool with that.

Reply:
I've addressed all of this.